samples/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ project(xlnt.samples)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Require C11 compiler
+set(CMAKE_C_STANDARD 11)
```

Comment:
idem

Reply:
See my other comment.

Reply:
I also removed the option here, since the samples only have 1 file containing C code (`std_image.h`).